### PR TITLE
fix more typos

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

**Review by maxkatz6 — APPROVED** [review]

File: tests/Avalonia.RenderTests/OpacityMaskTests.cs [hunk 1]
```diff
@@ -57,7 +57,7 @@ public async Task Opacity_Mask_Masks_Element()
         }
 
         [Fact]
-        public async Task RenderTansform_Applies_To_Opacity_Mask()
+        public async Task RenderTransform_Applies_To_Opacity_Mask()
```
Comment (maxkatz6) [inline tests/Avalonia.RenderTests/OpacityMaskTests.cs:60]: RenderToFile call below has [CallerMemberName] on its parameter. As well as CompareImages call. That's why tests are failing, as expected image files have different/old names.

**Review by maxkatz6 — APPROVED** [review]

File: tests/Avalonia.Markup.Xaml.UnitTests/Xaml/ControlTemplateTests.cs [hunk 2]
```diff
@@ -41,7 +41,7 @@ public void StyledProperties_Should_Be_Set_In_The_ControlTemplate()
 
                 window.ApplyTemplate();
                 button.ApplyTemplate();
-                var listBoxHierarchyLine = button.GetVisualChildren().ElementAt(0) as ListBoxHierachyLine;
+                var listBoxHierarchyLine = button.GetVisualChildren().ElementAt(0) as ListBoxHierarchyLine;
```
Comment (maxkatz6) [inline tests/Avalonia.Markup.Xaml.UnitTests/Xaml/ControlTemplateTests.cs:44]: ListBoxHierarchyLine was renamed in the C# code, but wasn't in the XAML code. This test is failing.

Comment (SimonCropp) [reply tests/Avalonia.Markup.Xaml.UnitTests/Xaml/ControlTemplateTests.cs:44]: good catch. fixed

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0028468-beta`. (feed url: https://pkgs.dev.azure.com/AvaloniaUI/AvaloniaUI/_packaging/avalonia-all/nuget/v3/index.json) [PRBUILDID]

**Pull request merged**